### PR TITLE
Fix computed seperator

### PR DESCRIPTION
**Pull request by bastianallgeier**
## Fixes

- https://github.com/getkirby/kirby/issues/3946

## Ready?

- [ ] Unit tests for fixed bug/feature
- [ ] In-code documentation (wherever needed)
- [ ] CI checks pass

## When merging

- [ ] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [x] Add changes to release notes draft in Notion

### CODE REVIEW

File: panel/src/components/Forms/Input/DateInput.vue [hunk 1]
```diff
@@ -149,7 +149,7 @@ export default {
      * Separator from `display` format
      */
     separator() {
-      return this.display.match(/[\W]/)[0];
+      return (this.display.match(/[\W]/) ?? [])[0] ?? "";
```
Comment (johannschopplich) [inline panel/src/components/Forms/Input/DateInput.vue]: `this.display.match(/[\W]/)?.[0] ?? ""` should work as well. 👀

Comment (afbora) [reply panel/src/components/Forms/Input/DateInput.vue]: Yes, this is more cleaner 👍

**Review by afbora — APPROVED** [review]

File: panel/src/components/Forms/Input/DateInput.vue (hunk 1 above)
Comment (bastianallgeier) [reply panel/src/components/Forms/Input/DateInput.vue]: @johannschopplich that looks a lot better indeed. I'm still not super familiar with the optional operator possibilities.

Comment (johannschopplich) [reply panel/src/components/Forms/Input/DateInput.vue]: @bastianallgeier That's why I wanted to gently point this out to you when I happened to look in the commit. Glad it to hear it looks good to you.

**Comment by distantnative** [comment]
Let's make sure to not do changes to Vue components while we are reworking them on another branch. Any PR should check if the change would be necessary there as well. I think we're good for this one, but just please keep in mind (otherwise we'll be reverting them unknowingly).

**Comment by afbora** [comment]
Is this a new issue? Previously it was only a problem when there was a conflict. I don't understand why the files we haven't changed are overwritten 🤔

**Comment by lukasbestle** [comment]
If the same line is changed in both PRs, Git will automatically trigger a conflict. Only issue is that it's then on us to resolve that conflict correctly. ;)

**Comment by distantnative** [comment]
The whole files will change pretty much, lines like these may move to totally different files. So well for sure have merge conflicts but it'd be very easy to overlook it then how to solve these properly. Better now, to remind ourselves to either not touch the Vue components at the moment at all or really we need to evaluate twice where to fix this in the refactored components too.

**Comment by lukasbestle** [comment]
Definitely best to avoid working on the same files in parallel if we can. 💯

**Pull request merged**